src/components/form/input.js

```diff
@@ -46,7 +46,7 @@ const StyledError = styled(CrossIcon)`
 
 const StyledInput = styled.input`
     background: none;
-    color: var(--color-${props => props.inputColor || 'black'});
+    color: var(--color- ${props => props.inputColor || 'black'});
```

Comment:
I think we need to add a prettier ignore line here else it will add the extra space after `--color- `

Reply:
I can see that there is already a space in my code!